modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java

```diff
@@ -371,12 +372,6 @@ private static void visitSchema(OpenAPI openAPI, Schema schema, String mimeType,
         }
     }
 
-    @FunctionalInterface
-    private static interface OpenAPISchemaVisitor {
-
-        public void visit(Schema schema, String mimeType);
-    }
-
```

Comment:
@martin-mfg do you know why this is removed? Is it because it's not being used at the moment?

(not saying we need to keep this)

cc @OpenAPITools/generator-core-team

Reply:
let's go ahead with this. we can always add it back later if there's a valid use case for this as it was added back in 2018.

Reply:
It's not really removed, but it is moved down to line 1964.